tests/xfstests/run.pm

```diff
@@ -502,6 +505,9 @@ sub run {
 
     config_debug_option;
 
+    # Load whitelist environment
+    my $env = prepare_whitelist_environment();
```

Comment:
$env is quite general to use in the main function run, it would be better to change it to $whitelist_env.

Reply:
replaced to $whitelist_env.